### DISCUSSION TITLE
Powerstrip Power Consumption showing zero

### DISCUSSION THREAD

**Discussion by fotis3d** [arendst/Tasmota#22456] [Support request]
Hello to all

I replaced the TYWE3S chip of a powerstrip I have, with 4relays (one of them is for the USB ports).
Placed an esp-12f and copied and pasted a working template.

I have 2 of those powerstrips and on the first one I cut the RX trace in order to flash it and on the second I replaced the chip. So I know the template I use is working fine cause the first one works fine. 

Now after I flashed and replaced the chip and pasted the same template all the relays work fine but I get no power consumption which the powerstrip supports.

It started giving me some numbers but after a while and some weird values (such as 2000v) it came down to zero and stayed there. So now everything is zero. 

Nevertheless the Energy seams to give values while nothing is connected so I suppose it is wrong.

I feel that 2 things could be the cause. Software (after all it is a new chip), of maybe I have not soldered very well the chip on the board (I am just a hobbist and it was my first time replacing a chip though I have flashed 40+ of my devices with Tasmota). But on the other hand how does it started giving values ?

Also I can not calibrate since there are no values. I tried of course but did nothing.

Any idea why is this happening or what I could do to fix this ? 

![image](https://github.com/user-attachments/assets/41649d25-286c-4c44-86de-0b9a3c66ebd1)

Thanks

**Comment by sfromis** [comment]
On Discord, vic already responded to your question:
> I'd re-check the solder joints. Bad connections can cause all kinds of weird behavior.

**Reply by fotis3d** [reply]
I was curious if there is anything I could try before I take a look again at the joints cause it is a pain to desmantle it again.
